static/js/market.js

```diff
@@ -215,15 +221,21 @@ const market = async () => {
 
         this.stalls = await Array.from(stalls.values())
 
-        this.products = Array.from(products.values()).map(obj => {
-          let stall = this.stalls.find(s => s.id == obj.stall_id)
-          obj.stallName = stall.name
-          obj.images = [obj.image]
-          if (obj.currency != 'sat') {
-            obj.formatedPrice = this.getAmountFormated(obj.price, obj.currency)
-          }
-          return obj
-        })
+        this.products = Array.from(products.values())
+          .map(obj => {
+            let stall = this.stalls.find(s => s.id == obj.stall_id)
+            if (!stall) return
```

Comment:
- the `return` inside a `.map()` means that the resulting array will have an `undefined` entry at that position
  - is this the desired outcome?